src/game_stats.py

```diff
@@ -28,11 +28,21 @@ def _send_discord_thread(self, msg: str):
             )
             send_discord_thread.daemon = True
             send_discord_thread.start()
-
+            
+    def _send_itemFound_discord_thread(self, msg: str):
+        if self._config.general["itemFound_discord_hook"]:
+            msg = f"{self._config.general['name']}: {msg}"
+            send_itemFound_discord_thread = threading.Thread(
+                target=send_discord,
+                args=(msg, self._config.general["itemFound_discord_hook"])
+            )
+            send_itemFound_discord_thread.daemon = True
+            send_itemFound_discord_thread.start()
```

Comment:
Please reuse the `_send_discord_thread()` function and make the discord hook an argument:
```python
def _send_discord_thread(self, msg: str, hook: str):
  if hook:
    msg = ...
```